sw/snRuntime/tests/dma_simple.c

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <dma.h>
```

Comment:
This include would be subsumed by the include of `snrt.h` if we go with the above suggestion.